pkg/kubectl/resource/builder.go

```diff
@@ -360,13 +396,33 @@ func hasCombinedTypeArgs(args []string) (bool, error) {
 	switch {
 	case hasSlash > 0 && hasSlash == len(args):
 		return true, nil
-	case hasSlash > 0 && hasSlash != len(args):
-		return true, fmt.Errorf("when passing arguments in resource/name form, all arguments must include the resource")
```

Comment:
What if it's "resource/name1 name2 name3",  will name2 and name3 be ignored silently?